controllers/utils.js

```diff
@@ -2,6 +2,8 @@
 
 const crypto = require("crypto");
 const uuidv4 = require("uuid/v4");
+const mozlog = require("../log");
+const log = mozlog("controllers.utils");
```

Comment:
Added debug logging. Information should show up in terminal when running locally.